build-caching/src/main/java/com/vertispan/j2cl/build/Project.java

```diff
@@ -15,6 +25,12 @@ public class Project implements com.vertispan.j2cl.build.task.Project {
 
     private boolean isJsZip = false;
 
+    private File jar;
+
+    private Optional<Boolean> isAPT = Optional.empty();
```

Comment:
IJ says you shouldn't have an Optional field... do they want an enum instead?  With more Javadoc on the interface, I don't think I care.